lib/sober_swag/output_object/field_syntax.rb

```diff
@@ -3,13 +3,19 @@ class OutputObject
     ##
     # Syntax for definitions that can add fields.
     module FieldSyntax
+      ##
+      # Defines a new field.
+      # @see SoberSwag::OutoutObject::Field#initialize
       def field(name, serializer, from: nil, &block)
         add_field!(Field.new(name, serializer, from: from, &block))
       end
 
       ##
       # Similar to #field, but adds multiple at once.
       # Named #multi because #fields was already taken.
+      #
+      # @param names [Array<Symbol>] the field names to add
+      # @param serializer [SoberSwag::Serializer::Base] the serializer to use for all fields.
       def multi(names, serializer)
         names.each { |name| field(name, serializer) }
       end
```

Comment:
https://github.com/SonderMindOrg/sober_swag/blob/better-rdoc-documentation/lib/sober_swag/output_object/field_syntax.rb#L31

Did we want the word **blueprint** to be here, or no?
Seems like we want another word.

Reply:
https://github.com/SonderMindOrg/sober_swag/blob/better-rdoc-documentation/lib/sober_swag/output_object/field_syntax.rb#L24

This phrasing doesn't make sense.